internal/api/ui/login/static/i18n/it.yaml

```diff
@@ -181,7 +181,7 @@ EmailVerification:
   Description: Ti abbiamo inviato un'e-mail per verificare il tuo indirizzo. Inserisci il codice nel campo sottostante.
   CodeLabel: Codice
   NextButtonText: Avanti
-  ResendButtonText: rispedisci
+  ResendButtonText: codice di reinvio
```

Comment:
```suggestion
  ResendButtonText: Reinvia
```

Reply:
its about mention the code in the button. that was somethin we found out in the user tests